Dockerfile.indigo

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM cartographer_ros:indigo
+
+# We require a GitHub access token to be passed.
+ARG github_token
+
+# First, we invalidate the entire cache if googlecartographer/cartographer_ros has
+# changed. This file's content changes whenever master changes. See:
+# http://stackoverflow.com/questions/36996046/how-to-prevent-dockerfile-caching-git-clone
+ADD https://api.github.com/repos/googlecartographer/cartographer_ros/git/refs/heads/master?access_token=$github_token \
+    cartographer_mir/cartographer_ros_version.json
+
+COPY . cartographer_mir
+
+RUN cartographer_mir/scripts/prepare_catkin_workspace.sh
```

Comment:
Seems like this file is not checked in. Also, can you re-use the one from cartographer_ros?
Same thing in the next lines and the other docker file.